tests/v1/entrypoints/llm/test_struct_output_generate.py

```diff
@@ -29,19 +30,20 @@
     "num_speculative_tokens": 5,
 }
 
-PARAMS_MODELS_BACKENDS_TOKENIZER_MODE = [
-    ("mistralai/Ministral-8B-Instruct-2410", "xgrammar", "auto", None),
-    ("mistralai/Ministral-8B-Instruct-2410", "guidance", "auto", None),
-    ("mistralai/Ministral-8B-Instruct-2410", "xgrammar", "mistral", None),
-    ("Qwen/Qwen2.5-1.5B-Instruct", "xgrammar", "auto", None),
+# yapf: disable
+PARAMS_MODELS_BACKENDS_TOKENIZER_MODE_REASONING_PARSER_SPEC_CONFIG = [
+    ("mistralai/Ministral-8B-Instruct-2410", "xgrammar", "auto", None, None),
+    ("mistralai/Ministral-8B-Instruct-2410", "guidance", "auto", None, None),
+    ("mistralai/Ministral-8B-Instruct-2410", "xgrammar", "mistral", None, None),
+    ("Qwen/Qwen2.5-1.5B-Instruct", "xgrammar", "auto", None, None),
+    ("deepseek-ai/DeepSeek-R1-Distill-Qwen-1.5B", "xgrammar", "auto", "deepseek_r1", None),
+    ("Qwen/Qwen3-0.6B", "xgrammar", "auto", "qwen3", None, NGRAM_SPEC_CONFIG),
+    ("mistralai/Ministral-8B-Instruct-2410", "guidance", "auto", None, NGRAM_SPEC_CONFIG),
     #FIXME: This test is flaky on CI thus disabled
-    #("Qwen/Qwen2.5-1.5B-Instruct", "guidance", "auto"),
-    ("mistralai/Ministral-8B-Instruct-2410", "guidance", "auto",
-     NGRAM_SPEC_CONFIG),
-    ("Qwen/Qwen2.5-1.5B-Instruct", "xgrammar", "auto", NGRAM_SPEC_CONFIG),
```

Comment:
and this test is now gone? I guess you changed it to Qwen3? I'd prefer to keep that type of change in a separate PR since it seems unrelated to thinking compatibility.

Reply:
I don't why I couldn't find it, haha